test/extended/authentication.sh

```diff
@@ -14,8 +14,6 @@ source "${OS_ROOT}/hack/common.sh"
 os::log::install_errexit
 cd "${OS_ROOT}"
 
-ensure_iptables_or_die
```

Comment:
> The tests can be run against an existing server, so this check is excessive.

This test doesn't seem to have gotten the news: https://github.com/openshift/origin/pull/5447/files#diff-cbfdb4abd12b4cc83e7ccde4c32d0281L42